src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -204,34 +200,25 @@ public Object instanceMain(final String[] argv) {
 
     /**
     *
-    * @return true if command line is valid
+    * @return true if program should be executed, false if an information only argument like {@link SpecialArgumentsCollection#HELP_FULLNAME} was specified
```

Comment:
I've changed the behavior of this method.  It now always throws if validation, including custom validation was unsuccessful.  I believe this is much more consistent behavior with the rest of the codebase.